fetools-app/src/components/ColorPicker/ColorPickerUtils.js

```diff
@@ -74,3 +74,35 @@ export function isValidColor(color){
 export function colorWithAlpha(color, alpha){
   return tinycolor(color).setAlpha(alpha).toHex8String()
 }
+
+export function createMonochromatic(currentColor){
+
+  const colors = tinycolor(currentColor).monochromatic(11);
+
+  const colorsHsl = colors.map((color)=> color.toHsl());
+
+  colorsHsl.sort((a,b)=> b.l-a.l)
+
+  return colorsHsl.map(color=>tinycolor(color).toHexString())
+}
+
+export function createAnalogous(currentColor){
+
+  const colors = tinycolor(currentColor).analogous(3,2);
+
+  return colors.filter((color,idx)=>idx===0?false:true).map(color=>color.toHexString())
+}
+
+export function createComplimentary(currentColor){
+
+  const colors = tinycolor(currentColor).splitcomplement();
+
+  return colors[1].toHexString()
+}
```

Comment:
This should be the direct complementary color, not one of the split complements.

```suggestion
export function createComplimentary(currentColor){

  const color = tinycolor(currentColor).spin(180);

  return color.toHexString()
}
```